Makefile

```diff
@@ -10,7 +10,7 @@ rebuild-bootstrapper:
 	docker build --no-cache -t ethereum-testnet-bootstrapper -f bootstrapper.Dockerfile .
 
 build-config:
-	docker build --build-arg="CONFIG_PATH=$(config)" -t etb-mainnet-config -f config.Dockerfile .
+	docker build -t etb-mainnet-config -f config.Dockerfile .
```

Comment:
I am using the build-arg for my dencun builds.

Reply:
okay I've restored it